qrhl/qrhl-input.el

```diff
@@ -0,0 +1,766 @@
+;;; qrhl-input.el --- Quail package for TeX-style input for qrhl-tool in ProofGeneral -*-coding: utf-8;-*-
+
+;; Copyright (C) 2001-2018 Free Software Foundation, Inc.
+;; Copyright (C) 2001, 2002, 2003, 2004, 2005, 2006, 2007, 2008, 2009,
+;;   2010, 2011
+;;   National Institute of Advanced Industrial Science and Technology (AIST)
+;;   Registration Number H14PRO021
+
+;; Author: TAKAHASHI Naoto <ntakahas@m17n.org>
+;;         Dave Love <fx@gnu.org>
+;; Keywords: multilingual, input, Greek, i18n
+
+;; This file is part of GNU Emacs.
+
+;; GNU Emacs is free software: you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+
+;; GNU Emacs is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with GNU Emacs.  If not, see <https://www.gnu.org/licenses/>.
+
+;;; Commentary:
+
+;; Modified by Dominique Unruh to adapt to qrhl-tool (original was latin-ltx.el)
+
+;;; Code:
+
+(if (version< emacs-version "26")
+    (error "Emacs version >= 26 required"))
```

Comment:
Question: do you already know what features would not be available with Emacs < 26?

Because FYI, proof-general is intended to be compatible with all emacs versions ≥ 25.1,
so if the PR was merged as is, these lines would contradict this claim:

https://github.com/ProofGeneral/PG/blame/master/README.md#L28-L36

So I'd like to suggest that we:
* test qrhl-input.el with emacs 25.1 (could you do this @dominique-unruh ?)
* if this work, just remove these two lines,
* otherwise if some features are missing, we could provide some "polyfill" (a complete "if-then-else")

Reply:
`qrhl-input.el` was based on `latin-ltx.el` from Emacs 26 which does not work on Emacs 25. (And the Emacs 25 version stopped working on Emacs 26.) Now I took the brute-force approach and simply made two files, one based on 25 and one on 26 (which also works in 27). 1787acf